src/commands/configure.ts

```diff
@@ -0,0 +1,30 @@
+import type { Run, Usage } from "../services/command/command.js";
+import { clearDefaultsConfig, promptDefaultsConfig } from "../services/config/defaults.js";
+import { println } from "../services/output/print.js";
+import { sprint } from "../services/output/sprint.js";
+
+export const usage: Usage = (_ctx) => {
+  return sprint`
+  Make changes to the configured defaults. This allows you to set an option on every ggt command by default without
+  needing to set a flag on every command.
+
+  {gray Usage}
+    ggt configure change
+    
+    ggt configure clear
+  `;
+};
+
+export const run: Run = async (ctx, args): Promise<void> => {
```

Comment:
Do you mind creating `spec/commands/configure.spec.ts` and adding some simple tests for this?

I'm certain this command will work as expected, but just to make sure it continues to work as expected in the future 😅 

For reference, here's the tests we have for the `whoami` command: https://github.com/gadget-inc/ggt/blob/main/spec/commands/whoami.spec.ts